modules/compiler/multi_llvm/include/multi_llvm/triple.h

```diff
@@ -16,12 +16,7 @@
 #ifndef MULTI_LLVM_TRIPLE_H_INCLUDED
 #define MULTI_LLVM_TRIPLE_H_INCLUDED
 
-#include <multi_llvm/llvm_version.h>
-
-#if LLVM_VERSION_MAJOR >= 17
 #include <llvm/TargetParser/Triple.h>
-#else
-#include <llvm/ADT/Triple.h>
-#endif
+#include <multi_llvm/llvm_version.h>
```

Comment:
This whole header was only so that users didn't need the `#if` for how to include `Triple.h`. Now that this is no longer needed, can we update users of `<multi_llvm/triple.h>` to include `<llvm/TargetParser/Triple.h>` directly?

Reply:
agreed. Done.